src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -204,7 +204,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             _optionCompareText = optionCompareText
             _embedVbCoreRuntime = embedVbCoreRuntime
             _suppressEmbeddedDeclarations = suppressEmbeddedDeclarations
-            _parseOptions = parseOptions
+            _parseOptions = If(parseOptions, VisualBasicParseOptions.Default) ' Required to correctly check for the presence of feature flags.
```

Comment:
>' Required to correctly check for the presence of feature flags. [](start = 78, length = 64)

Could you please elaborate why this is "Required to correctly check for the presence of feature flags."?